dist_run.py

```diff
@@ -182,11 +284,12 @@ def main():
 
     # Distribute model on TP mesh
     model.distribute(tp_mesh)
-    logger.info(f"Model: {model}")
+    # logger.info(f"Model: {model}")
 
-    mbs = 2  # number of micro-batches
+    mbs = 1  # number of micro-batches
```

Comment:
temporary code change? (not intended to commit?)

Reply:
this is intentional for now as just focused on single prompt case.  
will update after we get batch support for decoding.